packages/libs/client-utils/src/marmalade/create-token.ts

```diff
@@ -0,0 +1,68 @@
+import type {
+  ChainId,
+  IPactModules,
+  PactReference,
+  PactReturnType,
+} from '@kadena/client';
+import { Pact, readKeyset } from '@kadena/client';
+import {
+  addKeyset,
+  addSigner,
+  composePactCommand,
+  execution,
+  setMeta,
+} from '@kadena/client/fp';
+import { PactNumber } from '@kadena/pactjs';
+import { submitClient } from '../core/client-helpers';
+import type { IClientConfig } from '../core/utils/helpers';
+
+interface ICreateTokenInput {
+  policies?: string[];
+  uri: string;
+  tokenId: string;
+  precision: number;
+  chainId: ChainId;
+  creator: {
+    account: string;
+    keyset: {
+      keys: string[];
+      pred: 'keys-all' | 'keys-2' | 'keys-any';
+    };
+  };
+}
+
+const createTokenCommand = ({
+  policies = [],
+  uri,
+  tokenId,
+  precision,
+  creator,
+  chainId,
+}: ICreateTokenInput) =>
+  composePactCommand(
+    execution(
+      Pact.modules['marmalade-v2.ledger']['create-token'](
+        tokenId,
+        new PactNumber(precision).toPactInteger(),
+        uri,
+        policies.length > 0
```

Comment:
Does this create a valid policy array?

Reply:
Can you please add a unit test for these function? It's good to see what a valid command looks like